File: Models/ApplicationSystem.cs
using System.ComponentModel.DataAnnotations;

namespace AuthorizationForm.Models
{
    public class ApplicationSystem
    {
        public int Id { get; set; }
        
        [Required]
        public string Name { get; set; } = string.Empty;
        
        public string? Description { get; set; }
        public string? Category { get; set; }
        public bool IsActive { get; set; } = true;
        
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: php-version/app/Models/ApplicationSystem.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class ApplicationSystem extends Model
{
    protected $table = 'systems';

    protected $fillable = [
        'name',
        'description',
        'category',
        'is_active',
    ];

    protected $casts = [
        'is_active' => 'boolean',
        'created_at' => 'datetime',
        'updated_at' => 'datetime',
    ];
}

